docs/application/tizen-studio/setup/install-ubuntu.md

```diff
@@ -0,0 +1,181 @@
+# Installation in Ubuntu
+
+## Introduction
+This section guides you with downloading and installing Tizen Studio on your development hardware. 
+
+Follow the sub sections chronologically to get Tizen Studio installed with ease:
+
+- Pre-Installation check
+  - Operating System prerequisites
+  - Hardware prerequisites
+  - Software prerequisites
+- Setup OpenJDK in Ubuntu
+- Install dependencies
+- Launch Installer
+<!---- Troubleshooting 
+- Uninstalling -->
+
+## Pre-Installation Check 
+To successfully install Tizen Studio, ensure all the prerequisites are met.
+
+### Operating System Prerequisites
+Ensure that the target system meets the following operating system requirements:
+<table>
+  <tr>
+    <th colspan="2">Module </th>
```

Comment:
The page separation was too rough. This page is for Ubuntu, but Prerequisites information for Windows and MacOs are included.

Same issue to pages for macOS and Windows, too.